### ISSUE TITLE
Loc rows are all labeled with "en"

### ISSUE THREAD

**Issue by gryphonmyers** [piotr-cz/cockpit-import#5]
![image](https://cloud.githubusercontent.com/assets/5785304/13795751/d6d3b266-eabf-11e5-883f-b53aba2cb2b7.png)

I haven't tested whether this is merely a display issue or if it imports into the wrong rows as well

**Comment by piotr-cz** [comment]
There might be indeed a problem, because I've tested only scenario when default Cockpit language is Polish and additional is English.

By the way: if you add a languageCode suffix to column names in spreadsheet file, these should be auto-mapped. For example:

| productNo | name | name_en | description | description_en |
| --- | --- | --- | --- | --- |

**Comment by piotr-cz** [comment]
Okay, the problem is that additional language codes are [hardcoded](https://github.com/piotr-cz/cockpit-import/blob/v1.1.0/assets/js/index.js#L224) to `'en'` but should be `locale`.

So whole block should read

``` JS
localizations.push({
    code  : locale,
    output: '',
    filter: filter,
    column: undefined
});
```

Please confirm this is solution to the issue.

**Comment by gryphonmyers** [comment]
Yes that fixes the issue I was seeing. Want a PR or you got it?

**Comment by piotr-cz** [comment]
Fixed in a27a487030419e1cac98930d5471dde608fd78d5. Thanks for bugreport.
